### ISSUE TITLE
Npm support

### ISSUE THREAD

**Issue by leobalter** [timdown/rangy#274]
NPM modules are not only used for node, but also for browser files. You might agree it's interesting to change the Readme file in order to just mention it can also be installed through npm modules. 

The way it is presented today, it seems you can drop the npm support at anytime.

**Comment by timdown** [comment]
I confess I've done very little with Node.js, but I'm not quite sure I understand your point. Here is what I understand and would like to get across in the README:

- Rangy is on npm and I have no plans to remove it
- It would obviously be pointless to run Rangy in Node.js since it's a browser library
- I'm aware of Browserify and have tested Rangy with it

Is my understanding flawed? What else do you think I should be saying?

**Comment by leobalter** [comment]
In many projects I use NPM to fetch both node (mostly grunt plugins) and browser JS dependencies. 

Even when bower might a good option for those who wants only browser dependencies, npm have also other dependencies like the grunt plugins which leads me to use only it as a single package manager in my projects.

So, the same way I get jQuery from npm to run on my browser code, I want to be able to get Rangy. It's not pointless, I'm not trying to use it with node, but I'm downloading it through npm. Also, I'm not using Browserify on those projects, I'm just getting the files to be loaded on my browser directly from my html.

Basically, what I'm trying to say is that is great thing to have Rangy on npm. It's obviously not meant to run on Node, but that doesn't make less valuable for me to just use one package manager. Based on the readme text I wasn't confident you gonna give the same value to npm as bower when publishing new versions.

**Comment by timdown** [comment]
OK, thank you for explaining and I think I understand. I'll reword the README. The next thing I'm going to do is write a script to automate publishing a new version, so npm should always stay in sync.

**Comment by leobalter** [comment]
If you use [semver](http://semver.org/) you can basically run `npm version`, like in the example below:

`npm version 1.0.0`

It will update your version on package.json, commit and tag.

Reference: https://docs.npmjs.com/cli/version

You can also use https://github.com/mikaelbr/mversion to propagate the new version to the bower module.

**Comment by timdown** [comment]
Yes, I'd done a little research and come across those. Thanks.

**Comment by fresheneesz** [comment]
Sounds like this can be closed now

**Comment by leobalter** [comment]
Thank you, @timdown. Btw, the jQuery Plugins directory went to npm as well: http://plugins.jquery.com/
